powerlevel9k.zsh-theme

```diff
@@ -744,9 +744,11 @@ prompt_docker_machine() {
 # GO prompt
 prompt_go_version() {
   local go_version
+  local go_path
   go_version=$(go version 2>/dev/null | sed -E "s/.*(go[0-9.]*).*/\1/")
+  go_path=$(go env GOPATH)
 
-  if [[ -n "$go_version" ]]; then
+  if [[ -n "$go_version" && "${PWD##$go_path}" != "$PWD" ]]; then
```

Comment:
Do you want the segment to be printed if the GOPATH is empty? If the answer is yes, then this conditional needs to be changed. If no, then you should change the test to cover that.